deployments/daemon-set/nginx-plus-ingress.yaml

```diff
@@ -11,6 +11,9 @@ spec:
     metadata:
       labels:
         app: nginx-ingress
+     #annotations:
```

Comment:
I'd prefer to see a space between `#` and comment lines in yaml. This allows use of the ctrl+/ shortcut on text editors to comment/uncomment yaml config without breaking indentation levels.

Reply:
@Dean-Coakley  good observation. if anything, that would be a separate task

Reply:
I will keep like this for consistency, maybe we can change all of them in the future if that seems better. I am open to both ways (agree that maybe a space after the `#` is easier to read) thanks!